components/molecules/FilterHeader/filter-header.tsx

```diff
@@ -22,7 +22,7 @@ const HeaderFilter = () => {
   const { filterValues } = useFilterPrefetch();
   const { filterName, toolName, selectedFilter } = router.query;
   const filterBtnRouting = (filter: string) => {
-    captureAnayltics("Filters", "toolsFilter", `${filter} applied`);
+    captureAnayltics({ title: "Filters", property: "toolsFilter", value: `${filter} applied` });
```

Comment:
The identifier still needs to be wired up to all of these `captureAnalytics` calls.